core/templates/components/skill-selector/select-skill-modal.controller.ts

```diff
@@ -55,5 +55,8 @@ angular.module('oppia').controller('SelectSkillModalController', [
       $scope.confirm(totalSkills.find(
         summary => summary.id === $scope.selectedSkillId));
     };
+    $scope.setSelectedSkillId = function(event: string) {
```

Comment:
I think `skillId` would be a better name than `event`.
```suggestion
    $scope.setSelectedSkillId = function(skillId: string) {
```

Reply:
Done